examples/ota-provider-app/linux/main.cpp

```diff
@@ -271,6 +327,31 @@ int main(int argc, char * argv[])
     }
 
     chip::DeviceLayer::ConfigurationMgr().LogDeviceConfig();
+
+    if (gSetupDiscriminator.HasValue())
+    {
+        // Set discriminator to user specified value
+        ChipLogProgress(SoftwareUpdate, "Setting discriminator to: %" PRIu16, gSetupDiscriminator.Value());
+        err = chip::DeviceLayer::ConfigurationMgr().StoreSetupDiscriminator(gSetupDiscriminator.Value());
+        if (err != CHIP_NO_ERROR)
+        {
+            ChipLogError(SoftwareUpdate, "Setup discriminator setting failed with code: %" CHIP_ERROR_FORMAT, err.Format());
+            return 1;
+        }
+    }
+
+    if (gSetupPasscode.HasValue())
+    {
+        // Set passcode to user specified value
+        ChipLogProgress(SoftwareUpdate, "Setting passcode to: %" PRIu32, gSetupPasscode.Value());
+        err = chip::DeviceLayer::ConfigurationMgr().StoreSetupPinCode(gSetupPasscode.Value());
+        if (err != CHIP_NO_ERROR)
+        {
+            ChipLogError(SoftwareUpdate, "Setup passcode setting failed with code: %" CHIP_ERROR_FORMAT, err.Format());
+            return 1;
+        }
+    }
```

Comment:
Please remove passcode setting. This will soon be removed from the interface, since it is not spec compliant to store the passcode in the same processing unit as the PAKE verifier. In the meantime, leaving the discriminator setter is enough to disambiguate instances.

See https://github.com/project-chip/connectedhomeip/pull/14676 and #7738

Reply:
Thanks for pointing it out, removed passcode setting.